deploy-cluster-aws/awsdeploy.sh

```diff
@@ -116,7 +117,7 @@ fab upgrade_setuptools
 
 if [ "$WHAT_TO_DEPLOY" == "servers" ]; then
     # (Re)create the RethinkDB configuration file conf/rethinkdb.conf
-    python create_rethinkdb_conf.py
+    python create_rethinkdb_conf.py --bind-http-to-localhost $BIND_HTTP_TO_LOCALHOST
```

Comment:
exactly, `launch_ec2_nodes.py` will also fail if the key is not set so you can be confident that the key will always need to exist for the entire deployment to run